lambdas/access_counts/index.py

```diff
@@ -18,33 +21,107 @@
 ACCESS_COUNTS_OUTPUT_DIR = os.environ['ACCESS_COUNTS_OUTPUT_DIR']
 
 
+LAST_UPDATE_KEY = 'ObjectAccessLog.last_updated_ts'
```

Comment:
We could hit consistency issues here since we reuse the key name each rune... I guess the one hour cron spacing prevents that.

Reply:
Any reason not to use .txt as the extension? That way it will show in catalog, people can open it, etc

Reply:
Any reason not to use .txt as the extension? That way it will show in catalog, people can open it, etc.